save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/internal/UsersController.kt

```diff
@@ -81,6 +81,6 @@ class UsersController(
     }
 
     companion object {
-        private val roleForNewUser = listOf(Role.VIEWER).joinToString(",")
```

Comment:
this is weird. Also I still do not know how just "VIEWER" without ROLE prefix worked

Reply:
it's value for database entity, we will store it as `ROLE_VIEWER` in database.
I guess it can be fixed only on Spring level by changing `com.saveourtool.save.authservice.utils.SaveUserDetails#getAuthorities` -- here we create `SimpleGrantedAuthority` and need to add prefix there

Reply:
btw, checked -- old users have prefix `ROLE_` in database, the new ones -- don't

Reply:
Actually no, I see in permission validation endpoint that we check the value stored in Spring security with “as spring value” with ROLE_ prefix

Reply:
> So basically in role field it is much logical to write it with ROLE_ prefix

I'm not sure that it's more logical (we can read as enum from database and make logic on it), but historically we do it. Will fix existed values in prod database

Reply:
> Actually no, I see in permission validation endpoint that we check the value stored in Spring security with “as spring value” with ROLE_ prefix.

I see that we search a Role (our enum) using Spring security values (taking into account prefix) and then we validate permission using our Role only.

> also tested - mapping in the frontend does not work without a ROLE_ in a role field in a database

Yes, it's a bug. Will fix entries in database